epochX/cudacpp/CODEGEN/PLUGIN/CUDACPP_SA_OUTPUT/output.py

```diff
@@ -294,39 +297,100 @@ class PLUGIN_ProcessExporter_MadEvent(PLUGIN_ProcessExporter):
     from_template['SubProcesses'] = from_template['SubProcesses'] + [s+'gpu/fbridge_common.inc',
                                       s+'gpu/counters.cc',
                                       s+'gpu/ompnumthreads.cc']
-     
-    to_link_in_P = PLUGIN_ProcessExporter.to_link_in_P + ['fbridge_common.inc', 'counters.cc','ompnumthreads.cc'] 
+
+    to_link_in_P = PLUGIN_ProcessExporter.to_link_in_P + ['fbridge_common.inc', 'counters.cc','ompnumthreads.cc']
 
 #------------------------------------------------------------------------------------
 
 class SIMD_ProcessExporter(PLUGIN_ProcessExporter_MadEvent):
-    
+
     # Default class for the run_card to use
     run_card_class = launch_plugin.CPPRunCard
-    
+
     def change_output_args(args, cmd):
         """ """
-        cmd._export_format = "madevent"
+        #cmd._export_format = "madevent_forplugin"
+        cmd._export_format = 'madevent'
+        cmd._export_plugin = FortranExporterBridge
         args.append('--hel_recycling=False')
         args.append('--me_exporter=standalone_simd')
         if 'vector_size' not in ''.join(args):
             args.append('--vector_size=16')
+        if 'nb_wrap' not in ''.join(args):
+            args.append('--nb_wrap=1')
```

Comment:
Not sure what this is, but WARP not WRAP...

Reply:
Ok I will open an issue for this